src/main/java/com/shinobicoders/teamcodeapi/model/Project.java

```diff
@@ -0,0 +1,135 @@
+package com.shinobicoders.teamcodeapi.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.util.Date;
+import java.util.Objects;
+
+@Entity
+public class Project {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String name;
+
+    private String description;
+
+    private boolean status;
+
+    private int participantNumber;
```

Comment:
In our DB scheme it is "participants_number", but you have participantNumber without "s". I am afraid it can break ORM